.evergreen/config.yml

```diff
@@ -2931,7 +2931,7 @@ buildvariants:
   tasks:
      - name: "test-enterprise-auth"
 
-- matrix_name: "tests-mod-wsgi"
+- matrix_name: "tests-mod-wsgi-aws"
```

Comment:
Let's revert the name change to avoid loosing task history.

Reply:
Done